### PR TITLE
CircleCI integration tests: adding more APIs

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
The following are known to fail - I want to push them so that we can track them:

- dns
- bigquery
- logging

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
PTAL

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,37 +6,140 @@ anchor_docker: &anchor_docker
       environment:
         TERM: dumb
 
+anchor_auth_vars: &anchor_auth_vars
+  environment:
+    GCLOUD_PROJECT: gcloud-devel
+    GOOGLE_APPLICATION_CREDENTIALS: .circleci/it-service-account.json
```
Comment (andreamlin) [inline .circleci/config.yml]: Even though this is encrypted, is it ok to host in a public repo?

Comment (garrettjonesgoogle) [reply .circleci/config.yml]: I think so. Other languages are already doing it. (I know, great excuse, isn't it?)

File: utilities/verify_single_it.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ echo "----- building and installing parent pom -----"
 mvn -B -N install -DskipTests
 
 echo "----- building and installing shared modules -----"
-mvn -B -pl google-cloud-core,google-cloud-core-http,google-cloud-core-grpc install -DskipTests
+mvn -B -pl google-cloud-core,google-cloud-core-http,google-cloud-core-grpc,google-cloud-storage,google-cloud-pubsub install -DskipTests
```
Comment (andreamlin) [inline utilities/verify_single_it.sh]: why did storage and pubsub get put in this command?

Comment (garrettjonesgoogle) [reply utilities/verify_single_it.sh]: It was the easiest way to deal with the fact that bigquery depends on storage, and notification depends on pubsub (and storage).

**Pull request merged**